src/PoolManager.sol

```diff
@@ -175,25 +179,38 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, Claims {
     }
 
     modifier onlyByLocker() {
```

Comment:
now that the complexity of this modifier has increased, its probably worth having natspec for it